Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -3049,7 +3153,14 @@ void drawQueryBond1(MolDraw2D &d2d, const Bond &bond, bool highlight_bond,
       d2d.drawOptions().scaleBondWidth =
           d2d.drawOptions().scaleHighlightBondWidth;
     }
-    d2d.drawLine(at1_cds, at2_cds, col1, col2);
+    if (split) {
+      d2d.setActiveAtmIdx(bond.getBeginAtomIdx());
+      d2d.drawLine(at1_cds, midp, col1, col1);
+      d2d.setActiveAtmIdx(bond.getEndAtomIdx());
+      d2d.drawLine(midp, at2_cds, col2, col2);
+    } else {
+      d2d.drawLine(at1_cds, at2_cds, col1, col2);
+    }
```

Comment:
This bit of code ends up being repeated all over the place. Maybe it's worth pulling it out into a small function in the anonymous namespace?

Reply:
Refactored code to minimize repeated blocks.